<launch>

    <node pkg="prometheus_mission" type="ground_station" name="groud_station" output="screen">
        <param name="swarm_num" value="4"/>
        <param name="location_source" value="4"/>
        <param name="swarm_size" value="1.0"/>
    </node>

</launch>
